docs/contributors.yaml

```diff
@@ -231,6 +231,12 @@
   jira_user: drcrallen
   reviewboard_user: drcrallen
 
+- name: Charles Natali
+  emails:
+    - cf.natali@gmail.com
+  jira_user: charle
+  reviewboard_user: cf.natali
+
```

Comment:
Could you please move this into a separate commit (within this PR)?
Example: https://github.com/apache/mesos/commit/beaf0cd844f3658bfccb86049f7181036b0e6ae4

Reply:
Done!